man/systemd.unit.xml

```diff
@@ -750,7 +750,7 @@
         type when precisely a unit has finished starting up. Most importantly, for service units start-up is
         considered completed for the purpose of <varname>Before=</varname>/<varname>After=</varname> when all
         its configured start-up commands have been invoked and they either failed or reported start-up
-        success.</para>
+        success. Note that this does includes ExecStartPost (or ExecStopPost for the shutdown case).</para>
```

Comment:
enclose the settings in `<varname>` and suffix with `=` please

Reply:
Whops, my bad - follow-up: https://github.com/systemd/systemd/pull/15374